### PR TITLE
Cleanup from view conversion

### PR DESCRIPTION
**Pull request by ivan-BRHM**
tried to restore the file to the correct version, with all features intact. The program is running fine now.

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -134,7 +128,7 @@ def reset(self):
     def is_invincible(self):
         return self.invincibility_timer > 0
 
-    def on_update(self, delta_time: float = 1 / 60):
+    def update(self, delta_time: float = 1 / 60):
```
Comment (tobiasbp) [inline my_game.py:131]: 1: _on_update_ is the standard name for an update function which takes _delta_time_.
2: Having a default for _delta_time_ is weird. Why would one want that?

Comment (ivan-BRHM) [reply my_game.py:131]: yes, i think that having an on_update function for a sprite is bad practice in arcade...
So we might have to move the invincibility check to ingamewiev?

Comment (ivan-BRHM) [reply my_game.py:131]: right, it is not bad practise - thought sprite didn't have on_update.
However the function apparently needs us to fill in delta_time, so calling the function will look like this:
`# update player sprite
self.player_sprite.on_update(delta_time)`

Comment (tobiasbp) [reply my_game.py]: Yes, and that is what we want (We really need to do it everywhere). If we don't use delta time, the "speed" of the game will vary between computers.

Comment (tobiasbp) [reply my_game.py]: Please remove the _1/60_ default value for _delta_time_:
```
def on_update(self, delta_time)
```

**Comment by tobiasbp** [comment]
Please remove the _1/60_ default value for _delta_time_:
```
def on_update(self, delta_time)
```

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**